src/model/EthqlQuery.ts

```diff
@@ -121,6 +130,15 @@ class EthqlQuery {
   public health() {
     return 'ok';
   }
+
+  public queryContainsTxs(info: GraphQLResolveInfo): boolean {
+    const fields = getFieldList(info);
+    return (
+      !!fields.find(
+        a => a.includes('transactions') || a.includes('transactionsInvolving') || a.includes('transactionsRoles'),
```

Comment:
I suspect that the list of inner fields that triggers transaction retrieval should be a constant of some kind somewhere.

Reply:
SGTM - will change